download-data.py

```diff
@@ -244,12 +258,12 @@ def download_release_data(
 
     ### Update current link to point to new or test data if required ###
     # only do this if we are using test data or the specified release is "current" or "latest", not for specific dates
-    if update_current and not dryrun:
-        # update the current symlink
-        current_symlink = data_dir / "current"
-        current_symlink.unlink(missing_ok=True)
-        current_symlink.symlink_to(release)
-        print(f"Updated 'current' symlink to point to '{release}'.")
+    if update_current:
+        if not dryrun:
+            update_symlink(download_dir, release)
+        else:
+            print(f"\nIf this were not a dry run, the 'current' symlink would be updated to point to '{release}'.")
```

Comment:
I think we can be more compact here. 
```suggestion
            print(f"\nThe 'current' symlink would be updated to point to '{release}'.")
```